fasthttputil/inmemory_listener.go

```diff
@@ -1,11 +1,13 @@
 package fasthttputil
 
 import (
-	"fmt"
+	"errors"
 	"net"
 	"sync"
 )
 
+var errInmemoryListenerClosed = errors.New("InmemoryListener is already closed: use of closed network connection")
```

Comment:
```suggestion
var ErrInmemoryListenerClosed = errors.New("InmemoryListener is already closed: use of closed network connection")
```
Lets expose the type so people can check for it.

Reply:
Sure, done!